<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadImg</class>
 <widget class="QWidget" name="ReadImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>158</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="epsLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lastPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>上一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>下一页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>全屏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>正常模式</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>条漫模式</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="returePage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>returePage</tabstop>
  <tabstop>nextPage</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>lastPage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lastPage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>LastPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>NextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returePage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>ReturnPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>SwitchPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>SwitchPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LastPage()</slot>
  <slot>NextPage()</slot>
  <slot>ReturnPage()</slot>
  <slot>SwitchPicture()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
